From 6fade02acfac987837c928ef47ab7d380764297e Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Fri, 22 Jun 2012 13:15:49 +0100
Subject: [PATCH 22/38] EPEL 5: Remove Fedora MD test images.

On RHEL 5, mdadm is not able to create md devices with aliases,
eg. '/dev/md/boot'.
---
 tests/guests/Makefile.am |   12 ------------
 1 files changed, 0 insertions(+), 12 deletions(-)

diff --git a/tests/guests/Makefile.am b/tests/guests/Makefile.am
index f5570e9..3e8443f 100644
--- a/tests/guests/Makefile.am
+++ b/tests/guests/Makefile.am
@@ -47,8 +47,6 @@ check_DATA = \
 	blank-bootrootlv.img \
 	debian.img \
 	fedora.img \
-	fedora-md1.img \
-	fedora-md2.img \
 	ubuntu.img \
 	windows.img
 
@@ -70,16 +68,6 @@ fedora.img: guest-aux/make-fedora-img.pl \
 		guest-aux/fedora-packages.db
 	SRCDIR=$(srcdir) LAYOUT=partitions $(top_builddir)/run --test $<
 
-# Make a (dummy) Fedora image using md devices
-fedora-md1.img fedora-md2.img: stamp-fedora-md.img
-
-stamp-fedora-md.img: guest-aux/make-fedora-img.pl \
-		guest-aux/fedora-name.db \
-		guest-aux/fedora-packages.db
-	rm -f $@
-	SRCDIR=$(srcdir) LAYOUT=partitions-md $(top_builddir)/run --test $<
-	touch $@
-
 guest-aux/fedora-name.db: guest-aux/fedora-name.db.txt
 	rm -f $@ $@-t
 	mkdir -p guest-aux
-- 
1.7.4.1

